recipes-core/busybox/files/simple.script

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+# udhcpc script edited by Tim Riker <Tim@Rikers.org>
+# further modified by National Instruments
```

Comment:
Where is this script installed and used?

Reply:
Its installed as /etc/udhcpc.d/50default.
If the `-s` argument isn't given, udhcpc defaults to using [/usr/share/udhcpc/default.script](https://github.com/ni/openembedded-core/blob/nilrt/master/dunfell/meta/recipes-core/busybox/files/default.script#L1) which ends up executing /etc/udhcpc.d/50default

Reply:
Oh; I see. We're overwriting the upstream script implementation directly. OK.